ultralight-java-base/build.gradle

```diff
@@ -5,6 +5,14 @@ plugins {
 
 group 'com.labymedia'
 
+jar {
+	manifest {
+		attributes(
+			'Automatic-Module-Name': 'com.labymedia.ultralight'
+		)
+	}
+}

```

Comment:
please change the indentation to the 4 spaces used in the rest of the file. also goes for the databind build gradle

Reply:
Done